src/stores/contracts.ts

```diff
@@ -0,0 +1,68 @@
+import { derived, type Readable } from "svelte/store";
+
+import networks from "../../contracts/deployments/networks.json";
+
+import {
+  type Voting,
+  Voting__factory,
+  type TelediskoToken,
+  type ResolutionManager,
+} from "../../contracts/typechain";
+import { ResolutionManager__factory } from "../../contracts/typechain/factories/contracts/ResolutionManager/ResolutionManager__factory";
+import { TelediskoToken__factory } from "../../contracts/typechain/factories/contracts/TelediskoToken/TelediskoToken__factory";
```

Comment:
You can also import them from here in theory `../../contracts/typechain`, no need to go deep. (Probably autocomplete failed to resolve these deps using the shortest path.)